api/src/backend/api/models.py

```diff
@@ -1332,3 +1335,160 @@ class Meta:
                 statements=["SELECT", "INSERT", "UPDATE", "DELETE"],
             ),
         ]
+
+
+class LighthouseConfig(RowLevelSecurityProtectedModel):
```

Comment:
```suggestion
class LighthouseConfiguration(RowLevelSecurityProtectedModel):
```

Explicit is always better in model and table definitions.